clarity.d.ts

```diff
@@ -303,6 +312,27 @@ interface IClarityAssertFailedEventState extends IInstrumentationEventState {
 interface IClarityDuplicatedEventState extends IInstrumentationEventState {
   currentImpressionId: string;
 }
+
+interface IShadowDomInconsistentEventState extends IInstrumentationEventState {
+  // JSON of node indicies, representing the DOM
+  dom: NumberJson;
```

Comment:
>NumberJson [](start = 7, length = 10)

Nitpick: Did you mean to send back JSON specific to a node? If so, would NodeJson or something similar be a more descriptive name?